segway/gmtk/params.py

```diff
@@ -463,40 +634,68 @@ def __str__(self) -> str:
         if len(self) == 0:
             return ""
 
-        # The section generates the index and name of GMTK object
+        # if line_before is set, use it to begin the section's lines
+        lines = []
+        if self.line_before:
+            lines += [self.line_before]
+
+        # if stored items are generic strings, return them out without
+        # any additional formatting. Otherwise, apply formatting
+        if self.kind == OBJ_KIND_GENERICSTRING:
+            lines += self.get_unformatted_lines()
```

Comment:
Don't use `+=` with a list. Switch to `lines.extend()` or `lines.append()`.

Reply:
Changed throughout.